jp/components/grid_groupby.md

```diff
@@ -0,0 +1,140 @@
+﻿---
+title: Group By コンポーネント - ネイティブ  Angular | Ignite UI for Angular
+_description: Ignite UI for Angular Group By コンポーネントは、特定の列の共通の値に基づいてデータをグループ化します。
+_keywords: Ignite UI for Angular, UI コントロール, Angular ウィジェット, web ウィジェット, UI ウィジェット, Angular, ネイティブ Angular コンポーネント スイート, ネイティブ Angular コントロール, ネイティブ Angular コンポーネント ライブラリ, ネイティブ Angular コンポーネント, Angular Data Grid コンポーネント, Angular Data Grid コントロール, Angular Grid コンポーネント, Angular Grid コントロール, Angular 高パフォーマンス Grid, group by, groupby, グループ化, 列グループ化
+_language: ja
+---
+
+### グリッドのグループ化
+
+`igxGrid` は、列グループ値に基づいて階層グループのデータ レコードを可視化できる列グループをサポートします。グループは展開または縮小でき、グループのは UI または API で順序を変更できます。
+
+
+#### デモ
+
+<div class="sample-container loading" style="height:550px">
+    <iframe id="grid-sample-pinning-iframe" src='{environment:demosBaseUrl}/grid-sample-groupby' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="grid-sample-groupby-iframe" data-demos-base-url="{environment:demosBaseUrl}">stackblitz で表示</button>
+</div>
+
+#### 初期のグループ化状態
+
+グリッドの初期のグループ化状態を設定するには、式の配列をグリッドの filteringExpressions プロパティに設定します。
+
+```typescript
+public ngOnInit() {
+    grid.groupingExpressions = [
+        { fieldName: 'ProductName', dir: SortingDirection.Desc },
+        { fieldName: 'Released', dir: SortingDirection.Desc }
+    ];
+}
+```
+
+グループ式は、`ISortingExpression` インターフェイスを実装します。
+
+#### グループ化 API
+
+グループ化は、UI およびグリッド コンポーネントで公開された API で実行できます。各列の `groupable` プロパティを `true`.に設定してエンドユーザーは特定の列でグリッド データをグループ化できます。
```

Comment:
@yoshimis There is a `.` after `true` which I think should be replaced with a space.

Reply:
Removed.